zebra-consensus/src/primitives/ed25519.rs

```diff
@@ -120,36 +119,41 @@ impl Verifier {
 
     /// Flush the batch using a thread pool, and return the result via the channel.
     /// This function returns a future that becomes ready when the batch is completed.
-    fn flush_spawning(batch: BatchVerifier, tx: Sender) -> impl Future<Output = ()> {
+    async fn flush_spawning(batch: BatchVerifier, tx: Sender) {
+        let (rsp_tx, rsp_rx) = tokio::sync::oneshot::channel();
+
         // Correctness: Do CPU-intensive work on a dedicated thread, to avoid blocking other futures.
-        tokio::task::spawn_blocking(|| {
```

Comment:
Using `spawn_fifo` directly to avoid idle blocking threads counting towards `max_blocking_threads` (Zebra uses the default of 512).

Reply:
This seems like something important to put in the docs?

Reply:
It turns out this was irrelevant, Zebra doesn't seem to approach anywhere near the limit except briefly at startup.